packages/shorebird_cli/lib/src/commands/patch/ios_framework_patcher.dart

```diff
@@ -51,6 +52,30 @@ class IosFrameworkPatcher extends Patcher {
   @override
   double? get linkPercentage => lastBuildLinkPercentage;
 
+  /// The value of --split-debug-info-path if specified.
+  String? get splitDebugInfoPath =>
+      argResults[CommonArguments.splitDebugInfoArg.name] as String?;
+
+  // TODO(felangel): make this dynamic based on the platform and arch.
+  /// The name of the split debug info file.
+  static const splitDebugInfoFileName = 'app.ios-arm64.symbols';
```

Comment:
Do we need to throw if we ever try to use this method on the wrong platform?

Reply:
I believe it works as is on other platforms and this additional logic is only needed when we use the linker (only when patching ios and ios_framework)

Reply:
added e2e tests